aws/offsite/clouformation/verify-database.yml

```diff
@@ -0,0 +1,155 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  DatabasePasswordParameter:
+    Type: String
+    Description: Specify a new password for the database copy to use when verifying it.
+  SlackWebHookURLParameter:
+    Type: String
+    Description: >-
+      Slack Webhook URL for notifications such as
+      ('T039SAH7W/B85EJM39N/4EPCUIZVLWm6Tq0WgA8Xrt31')
```

Comment:
I think this looked OK to me but just to double-check, can you confirm this identifier a test/dummy endpoint that won't expose any sensitive/internal channels?